dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt

```diff
@@ -1,18 +1,21 @@
+# Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+# For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+
 btnIgnore = Label for the Ignore button of the Spell Check dialog window.
 btnIgnoreAll = Label for the Ignore All button of the Spell Check dialog window.
-btnReplace = 
-btnReplaceAll = 
-btnUndo = 
```

Comment:
It's fine that you removed redundant spaces, but this issue is very specific and it wasn't in its description. Now I don't know if it breaks anything? I think it should have separate PR.

Reply:
Actually it was automatically removed by my editor. I'll undo these changes.